### PR TITLE
Add support for user count over time

### PR DESCRIPTION
**Pull request by jmfiola**
This MR adds the "user_count" field to all locust requests

### CODE REVIEW

**Comment by jmfiola** [comment]
@pjcalvo FYI

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
```
Comment (pjcalvo) [inline setup.py]: I will increase the version when releasing. So this PR should just focus on the change.

File: example/requirements.txt [hunk 2]
```diff
@@ -1,3 +1,3 @@
-locust_influxdb_listener == 0.0.8
+locust_influxdb_listener == 0.0.9
```
Comment (pjcalvo) [inline example/requirements.txt]: Please also remove the version here.

**Review by pjcalvo — CHANGES_REQUESTED** [review]

File: setup.py (hunk 1 above)
Comment (jmfiola) [reply setup.py]: reverted

File: example/requirements.txt (hunk 2 above)
Comment (jmfiola) [reply example/requirements.txt]: reverted

**Comment by jmfiola** [comment]
> 

Done! No problem, happy to contribute.

**Comment by jmfiola** [comment]
@pjcalvo Looking for another review here

**Review by pjcalvo — APPROVED** [review]

**Comment by pjcalvo** [comment]
Thanks, sorry it took me a while to review it.

**Pull request merged**